File: Java/SETPractice/src/main/java/SOLIDPrinciple/InterfaceSegregation/Manager.java
package SOLIDPrinciple.InterfaceSegregation;

public class Manager implements ILead{
    @Override
    public void CreateTask() {
        System.out.println("Manager created task");
    }

    @Override
    public void AssignTask() {
        System.out.println("Manager assigned task");
    }
}

File: DotNet/SETPractice/SOLIDPrinciples/InterfaceSegregation/Manager.cs
namespace SOLIDPrinciples.InterfaceSegregation
{
    public class Manager : ILead
    {
        public void AssignTask()
        {
            Console.WriteLine("Manager assign task");
        }

        public void CreateTask()
        {
            Console.WriteLine("Manager create task");
        }
    }
}
